Description: Fix hyphen-used-as-minus-sign problem in manpages
Author: Jaromír Mike¨ <mira.mikes@seznam.cz>
Forwarded: No
---
 man/de/qmidiroute.1 |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- qmidiroute.orig/man/de/qmidiroute.1
+++ qmidiroute/man/de/qmidiroute.1
@@ -8,7 +8,7 @@
 .\"
 .TH QMIDIROUTE 1 2009-09-13
 .SH NAME
-qmidiroute \- MIDI-Signalverarbeiter und -Router
+qmidiroute \- MIDI-Signalverarbeiter und \-Router
 .SH SYNTAX
 .br
 .B qmidiroute 
@@ -22,7 +22,7 @@ qmidiroute \- MIDI-Signalverarbeiter und
 }
 .SH BESCHREIBUNG
 .B qmidiroute
-ist ein vielseitiger MIDI-Signalverarbeiter und -Router für den ALSA
+ist ein vielseitiger MIDI-Signalverarbeiter und \-Router für den ALSA
 Sequencer.  Die grafische Oberfläche basiert auf der Qt4 Bibliothek.
 .B qmidiroute
 erlaubt die Konfiguration einer beliebigen Anzahl von MIDI-Routen, in
